bin/inx-chronicle/src/util.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright 2022 IOTA Stiftung
```

Comment:
Consider naming this module `health.rs` or `health_check.rs` which is more specific.